.travis.yml

```diff
@@ -1,31 +1,37 @@
 language: php
 
+dist: trusty
+
 env:
   global:
-    - COMPOSER_ROOT_VERSION=2.0.x-dev
+    - COMPOSER_ROOT_VERSION=4.0.x-dev
```

Comment:
My understanding is that this should match the module, rather than core

@tractorcow ?

Reply:
Yes, this is simply the alias for "branch of the root project" which in this case is the module being checked out.